indigo/analysis/work_detail/base.py

```diff
@@ -19,15 +19,21 @@ class BaseWorkDetail(LocaleBasedMatcher):
     """ These numbers don't have numbered titles. """
     number_must_be_digit_doctypes = ['act']
     """ These doctypes only have numbered titles if the number starts with a digit."""
+    chapter_names_choices = (
+        ("chapter", _("Chapter")),
+    )
+    """ Subclasses can give choices in the Work form which will be used in the numbered title. """
 
     def work_numbered_title(self, work):
         """ Return a formatted title using the number for this work, such as "Act 5 of 2009".
         This usually differs from the short title. May return None.
         """
         # check chapter first
-        if work.place.settings.uses_chapter and work.properties.get('cap'):
-            # eg. Chapter 2
-            return _('Chapter %(cap)s') % {'cap': work.properties['cap']}
+        chapter_number = self.work_chapter_number(work)
+        if chapter_number:
+            return _('%(name)s %(number)s') % {
```

Comment:
This doesn't really make sense as a translation string -- the translator just sees two variables. Instead, since the name is already translated, just use a normal f-string.